Updated the organisation section on cloud with some more information (#4470)
umbraco/UmbracoDocs: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md

In the **Payment methods** section, you can view the payment methods for your organization. From here, you can add or delete credit card details for your Organization.

![Payment methods](images/payment-methods-v10.png)

## Payment History

> In the **Payment History** section, you can see the payment history for your organisation.

Reviewer: ```suggestion
In the **Payment History** section, you can see the payment history for your organization.
```

## Subscriptions

> In the **Subscriptions** section you can see the current active subscribtions that is running under the Umbraco Cloud organisation.

Reviewer: ```suggestion
In the **Subscriptions** section, you can see the current active subscriptions that are running under the Umbraco Cloud organization.
```
